src/dmd/statement.d

```diff
@@ -1591,6 +1591,12 @@ extern (C++) final class SwitchStatement : Statement
 
     final bool checkLabel()
     {
+        /*
+        * Checks the scope of a label for existing variable declaration.
+        * Params:
+        *   vd = variable declaration to check
+        * Returns: `true` if the variables declared in this label would be skipped.
```

Comment:
`grep`ping dmd source code revealed the prevailing style is to put the `Returns` description on the following line.

Reply:
I like that too, but Walter's comment was:

> These blank lines unnecessarily take up vertical space.

Reply:
😆  My comment was on Walter's behalf.  Ok, then I'm good.